packages/demo/src/app/home/home.component.ts

```diff
@@ -182,14 +182,14 @@ const regexReplace = `// Sass Functions. Don't forget to add the corresponding @
 // REPLACE WITH
 $1$2.$3 // Add this import to the top of the file: @use "sass:$1";
 
-// Variables, mixins and functions used from CWF. Be careful! This might replace more than you'd want it to...
+// Variables, mixins and functions used from post. Be careful! This might replace more than you'd want it to...
```

Comment:
```suggestion
// Variables, mixins and functions used from Swiss Post Design System. Be careful! This might replace more than you'd want it to...
```